builder/docker/docker.go

```diff
@@ -0,0 +1,337 @@
+package docker
+
+import (
+	"archive/tar"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Masterminds/cookoo"
+	"github.com/Masterminds/cookoo/log"
+	"github.com/Masterminds/cookoo/safely"
+	"github.com/deis/deis/builder/etcd"
+	docli "github.com/fsouza/go-dockerclient"
+)
+
+// Path to the Docker unix socket.
+var DockSock = "/var/run/docker.sock"
```

Comment:
The reason it's declared, and declared as a var, is so that it can be changed globally at startup time. (flag, net/http and other packages do this). But... I liked the idea of using the Docker-defined one as the default value (`DockSock = opts.DefaultUnixSocket`), but... it appears that that var doesn't exist in our copy of that repo. We're too far back. So... I checked go-dockerclient, and it has that same variable declared, but... it's not exported (`dockerUnixSocket`). Grr...

So... I guess I'll leave it for now, and fix it when we upgrade to a more recent Docker library.